assets/charts/control-plane/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -74,56 +40,6 @@ spec:
       - name: kube-apiserver
         image: {{ .Values.apiserver.image }}
         command:
-        {{- if eq (int .Values.apiserver.replicas) 1 }}
```

Comment:
We need to use `--permit-port-sharing=true` flag now.

Reply:
Actually, we should also add this flag to bootstrap kubelet and make sure they both run as the same user, so we can port-share between them too. This will allow to enable `--wait` in bootkube, which should provide more stable bootstrapping.

Reply:
Do you mean bootstrap kubeapiserver?

Reply:
Yes, sorry. bootstrap kube-apiserver of course :+1:

Reply:
This is added.